### DISCUSSION TITLE
areamap.nc using 5 arcmin

### DISCUSSION THREAD

**Discussion by weixiaoxun** [iiasa/CWatM#138] [Q&A] [answered]
Dear Professor @mikhailsmilovic ! 

Guten Morgen！I'm using CWatM 5arcmin for my experiment this time. When I tried to crop my mask area as I did before with 30min resolution, I couldn't find a file like "cellarea5.NC" (cellarea30) in the routing folder. I only see the "ldd.NC" file. Could you please provide me with the "cellarea5.NC" file? Thank you.

Best wishes
xiaoxun

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Tag @weixiaoxun, 
cellarea.nc is in [cwatm_input_5min](https://drive.google.com/drive/folders/1HqcBj5fD6DHJpOe-t_6JHFMKFtubghZf?usp=drive_link)/landsurface/topo 🟪

**Reply by weixiaoxun** [reply]
Thank you very much,Professor mikhailsmilovic!

Best wishes!
xiaoxun
